function-images/tests/eventing/README.md

```diff
@@ -0,0 +1,62 @@
+# Knative Eventing Test Images
+
+## Design
+We present a very simple _hello world_ pipeline where
```

Comment:
this doc should give a brief overview of the k8s manifests. Right now, I don't understand how they are organized. Can we keep all of them in a single file instead of many small files?